frontend/src/services/Wp.d.ts

```diff
@@ -0,0 +1,25 @@
+declare module Wp {
```

Comment:
Perhaps the better name would be `VpApi`? Seems very specific to VersionPress and its API.